### DISCUSSION TITLE
Zero to Hero ControlNet Extension Tutorial - Easy QR Codes - Generative Fill (inpainting / outpainting) - 90 Minutes - 74 Video Chapters - Tips - Tricks - How To

### DISCUSSION THREAD

**Discussion by FurkanGozukara** [Mikubill/sd-webui-controlnet#1716] [Show and tell]
Worked for days for this Tutorial,

I hope you add this tutorial to the wiki and relevant information sections

[**Zero to Hero ControlNet Tutorial: Stable Diffusion Web UI Extension | Complete Feature Guide**](https://youtu.be/3E5fhFQUVLo)

[![image](https://s3.amazonaws.com/moonup/production/uploads/6345bd89fe134dfd7a0dba40/OU_mG6YvrSOAZLboBwauu.png)](https://youtu.be/3E5fhFQUVLo)

**Comment by huchenlei** [comment]
That is a really good tutorial! Thanks for making this. @lllyasviel says he was on the documentation here: https://github.com/Mikubill/sd-webui-controlnet/issues/1093. Maybe let him know and consider put a link in the documentation/wiki page.

**Reply by FurkanGozukara** [reply]
Thanks. it took days to prepare this. so many options and features to cover :D
